test/test_en_middle_endian.js

```diff
@@ -1,21 +1,21 @@
 
 test("Test - Single Expression", function() {
-    var text = 'She is getting married next year (July 2017).';
+    var text = 'She is getting married next year (July 2013).';
```

Comment:
Instead of updating the test here, I prefer you remove "next year". And add a new test case for "next year (July 2013)" in test_en_relative.js